docs/sources/shared/datasources/tempo-traces-to-profiles.md

```diff
@@ -25,9 +25,13 @@ When configured, this connection lets you run queries from a trace span into the
 
 There are two ways to configure the trace to profiles feature:
 
-- Use a simplified configuration with default query, or
+- Use a basic configuration with default query, or
 - Configure a custom query where you can use a template language to interpolate variables from the trace or span.
 
+{{< admonition type="note">}}
+Traces to profile requires a configured Tempo data source with Traces to profiles enabled and a Pyroscope data source. This integration supports profile data generated using Go, Ruby, and Java instrumentation SDKs.
```

Comment:
We removed the feature toggles.

Reply:
How does removing the feature flags in 10.3 impact someone using Grafana Cloud? This file is shared amongst several products. 

I changed enabled to configured. Users still have to update the Tempo data source with the Traces ot profiles section, right? 
```suggestion
Traces to profile requires a Tempo data source with Traces to profiles configured and a Pyroscope data source. This integration supports profile data generated using Go, Ruby, and Java instrumentation SDKs.
```